lib/modules/manager/pixi/extract.ts

```diff
@@ -0,0 +1,58 @@
+import type { z } from 'zod';
+import { logger } from '../../../logger';
+import { getSiblingFileName, localPathExists } from '../../../util/fs';
+import { Result } from '../../../util/result';
+import type { PackageFileContent } from '../types';
+import { type PixiConfigSchema, PixiToml, PyprojectToml } from './schema';
+
+function getUserPixiConfig(
+  content: string,
+  packageFile: string,
+): null | z.infer<typeof PixiConfigSchema> {
```

Comment:
```suggestion
): null | PixiConfigSchema {
```
Infer this in `schema.ts`